package.json

```diff
@@ -75,5 +75,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
```

Comment:
This is the only thing I'm not sure about, should it be here or in `devDependencies` instead? Everything else is in `devDependencies`, not sure why.